src/lib/Repository/UiVersionService.php

```diff
@@ -30,11 +31,38 @@ public function __construct(ContentService $contentService)
      */
     public function deleteVersion(int $contentId, int $versionNo)
     {
-        $versionInfo = $this->contentService->loadVersionInfo(
-            $this->contentService->loadContentInfo($contentId),
+        $versionInfo = $this->loadVersionInfo(
+            $this->loadContentInfo($contentId),
             $versionNo
         );
 
         $this->contentService->deleteVersion($versionInfo);
     }
+
+    /**
+     * Creates a new draft based on the content and verisionNo.
+     *
+     * @param mixed $contentId
+     * @param mixed $versionNo
+     */
+    public function createDraft($contentId, $versionNo)
+    {
+        $contentInfo = $this->loadContentInfo($contentId);
+        $versionInfo = $this->loadVersionInfo(
+            $this->loadContentInfo($contentId),
```

Comment:
You can use `$contentInfo` from above. And I don't think it's worth wrapping that line then.